components/eam/src/chemistry/modal_aero/modal_aero_drydep.F90

```diff
@@ -733,4 +615,380 @@ subroutine calcram(ncol,landfrac,icefrac,ocnfrac, &
 
   end subroutine calcram
 
+  !==========================================================================
+  ! Calculate particle velocity of gravitational settling
+  !==========================================================================
+  subroutine modal_aero_gravit_settling_velocity( moment, ncol, pcols, nver,           &! in
+                                                  gravit, rair, pi, radius_max,        &! in
+                                                  tair, pmid,                          &! in
+                                                  radius_part, density_part, sig_part, &! in
+                                                  vlc_grv                              )! out
+
+    ! Arguments
+
+    integer,  intent(in) :: moment       ! moment of size distribution (0 for number, 2 for surface area, 3 for volume)
+    integer,  intent(in) :: ncol         ! # of grid columns to do calculations for
+    integer,  intent(in) :: pcols        ! dimension size (# of columns) 
+    integer,  intent(in) :: nver         ! dimension size (# of model layers) 
+
+    real(r8), intent(in) :: gravit            ! gravitational acceleration, [kg/m2/s]
```

Comment:
Units should be [m/s^2]

Reply:
Good catch. This line has been removed when removing `gravit` from argument lists.